threeML/plugins/FermiLATLike.py

```diff
@@ -482,33 +540,36 @@ def get_log_like(self):
     def __reduce__(self):
 
         return (
-            FermiLATUnpickler(),
+            FermiLATUnpickler(), 
             (
-                self.name,
-                self.eventFile,
-                self.ft2File,
-                self.livetimeCube,
+                self.name, 
+                self._event_file, 
+                self._ft2_file, 
+                self._livetime_cube_file, 
                 self.kind,
-                self.exposureMap,
-                self.likelihoodModel,
-                self.fit_nuisance_params,
+                self._exposure_map_file, 
+                self.likelihood_model, 
+                self.fit_nuisance_params
             ),
         )
-
-    # def __setstate__(self, likelihoodModel):
+    
+    # def __setstate__(self, likelihood_model):
     #
     #     import pdb;pdb.set_trace()
     #
-    #     self.set_model(likelihoodModel)
+    #     self.set_model(likelihood_model)
 
-    def getModelAndData(self):
+    def get_model_and_data(self):
 
         fake = numpy.array([])
 
         return fake, fake, fake, fake
 
     pass
 
+    def get_observation_duration(self):
```

Comment:
perhaps make a property?

Reply:
I am not sure a property here would work. We really don't  want this to be set from the user. So I change the variable name to __observation_duration so it is now protected.